tests/robustness/coverage/pattern_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2025 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package coverage_test
+
+import (
+	"strings"
+	"testing"
+)
+
+func normalize(k string) ([]string, string) {
```

Comment:
Please define a struct to represent normalized key.

Reply:
It is processed in only one place and it is extremely simple. Removed it instead